upstash/redis/database/data_source.go

```diff
@@ -48,7 +48,7 @@ func DataSourceDatabase() *schema.Resource {
 			"tls": &schema.Schema{
 				Type:        schema.TypeBool,
 				Computed:    true,
-				Description: "When enabled data is encrypted in transit",
+				Description: "When enabled, data is encrypted in transit. TLS is enabled by default for newly created databases and cannot be disabled.",
```

Comment:
No need to add the extra sentence I believe, since the parameter does not make any sense if not configurable anyways. We could remove tls alltogether, but let's not complicate for now

Reply:
Deprecated:  "Multizone option is deprecated. It is enabled by default for paid databases.",

we could use a similar addition for tls instead